lte/image_decode.py

```diff
@@ -0,0 +1,22 @@
+"""
+Decodes images from JPEG bytes to numpy array.
+"""
+
```

Comment:
Nit: Can we create a directory called `modules` and then put the `image_decode.py` file in that directory?